programming/python/libraries.md

```diff
@@ -11,43 +11,41 @@ Robot Kit
 
-* [astoria 0.8.0](https://pypi.org/project/astoria)
+* [astoria 0.10.0](https://pypi.org/project/astoria)
 * [cached-property 1.5.2](https://pypi.org/project/cached-property)
-* [click 8.0.1](https://pypi.org/project/click)
-* [cycler 0.10.0](https://pypi.org/project/cycler)
-* [dateutil 2.8.2](https://pypi.org/project/python-dateutil/)
-* dateutil-zoneinfo 2.8.2
+* [click 8.1.2](https://pypi.org/project/click)
+* [cycler 0.11.0](https://pypi.org/project/cycler)
+* [dateutil 2.8.2](https://pypi.org/project/python-dateutil)
 * [dbus-next 0.2.3](https://pypi.org/project/dbus-next)
 * [docopt 0.6.2](https://pypi.org/project/docopt)
-* [gmqtt 0.6.10](https://pypi.org/project/gmqtt)
-* [gpg 1.16.0](https://pypi.org/project/gpg)
+* [gmqtt 0.6.11](https://pypi.org/project/gmqtt)
+* [gpg 1.17.1](https://pypi.org/project/gpg)
 * [iniparse 0.5](https://pypi.org/project/iniparse)
-* [j5 0.13.1](https://pypi.org/project/j5)
-* [j5-zoloto 0.2.0](https://pypi.org/project/j5-zoloto)
-* [kiwisolver 1.3.2](https://pypi.org/project/kiwisolver)
-* [matplotlib 3.4.1](https://pypi.org/project/matplotlib)
-* [numpy 1.21.2](https://pypi.org/project/numpy)
-* opencv 4.5.2
-* [pandas 1.3.3](https://pypi.org/project/pandas)
-* [pillow 8.3.2](https://pypi.org/project/Pillow/)
-* pkg-resources 57.4.0
-* [pydantic 1.8.2](https://pypi.org/project/pydantic)
-* [pygobject 3.40.1](https://pypi.org/project/PyGObject/)
-* [pyparsing 2.4.7](https://pypi.org/project/pyparsing)
+* [j5 1.1.2](https://pypi.org/project/j5)
+* [j5-zoloto 0.4.0](https://pypi.org/project/j5-zoloto)
+* [kchd 0.4.0](https://pypi.org/project/kchd)
+* [matplotlib 3.5.1](https://pypi.org/project/matplotlib)
+* [numpy 1.22.3](https://pypi.org/project/numpy)
+* opencv 4.5.5
+* [pandas 1.4.2](https://pypi.org/project/pandas)
+* [pillow 9.0.1](https://pypi.org/project/Pillow)
+* [pydantic 1.9.1](https://pypi.org/project/pydantic)
+* [pygobject 3.42.0](https://pypi.org/project/PyGObject)
+* [pyparsing 3.0.7](https://pypi.org/project/pyparsing)
 * [pyquaternion 0.9.9](https://pypi.org/project/pyquaternion)
 * [pyserial 3.5](https://pypi.org/project/pyserial)
-* [pytz 2021.1](https://pypi.org/project/pytz)
-* [pyudev 0.22.0](https://pypi.org/project/pyudev)
+* [pytz 2022.1](https://pypi.org/project/pytz)
+* [pyudev 0.23.2](https://pypi.org/project/pyudev)
 * [pyusb 1.2.1](https://pypi.org/project/pyusb)
-* [pyyaml 5.4.1](https://pypi.org/project/PyYAML/)
-* [rpm 4.16.1.3](https://pypi.org/project/rpm)
-* [setuptools 57.4.0](https://pypi.org/project/setuptools)
+* [pyyaml 6.0](https://pypi.org/project/PyYAML)
+* [rpi.gpio 0.7.0](https://pypi.org/project/rpi.gpio)
+* [setuptools 59.5.0](https://pypi.org/project/setuptools)
 * [six 1.16.0](https://pypi.org/project/six)
-* [sr-robot3 2022.0.2](https://pypi.org/project/sr.robot3/)
-* [toml 0.10.2](https://pypi.org/project/toml)
+* [sr.robot3 2023.0.4](https://pypi.org/project/sr.robot3)
```

Comment:
I believe we bumped this to `.5` to fix the wheel dependency issue

Reply:
Looks like its still `.4` in `robot-image` https://github.com/srobo/robot-image/tree/main/sources/meta-srobo/recipes-robot/python-sr-robot3

Reply:
I thought `.5` only fixed a Windows issue?

Reply:
If `.4` is in the image, then yeah let's keep it as-is. Wasn't sure if a new image had been baked (yes it only fixes an issue on non-linux).